README.md

```diff
@@ -6,16 +6,18 @@ The current version assumes you're already generating power.log, which you are i
 
 To build, install Go, then:
 
-* Clone this repo to `$GOROOT/github.com/frogstack/HSTeamPlay/`
-* Run `go get github.com/ChrisHines/GoSkills/skills`
-* Run `go build` in `$GOROOT/github.com/frogstack/HSTeamPlay/`
+* Clone this repo to `$GOROOT/src/github.com/frogstack/HSTeamPlay/`
+* Run `go get github.com/ChrisHines/GoSkills/skills`. If this does not work, clone it to `$GOROOT/src/ChrisHines/GoSkills/skills/`
```

Comment:
You are right. Everything is in "$GOROOT/src/github.com" on my laptop. Sorry for that! I fixed the typo.